src/services/issuer.rs

```diff
@@ -366,12 +363,13 @@ pub fn create_revocation_status_list(
         let tails_reader = TailsFileReader::new_tails_reader(&rev_reg_def.value.tails_location);
         let issued = (1..=max_cred_num).collect::<BTreeSet<_>>();
 
-        CryptoIssuer::update_revocation_registry(
+        Issuer::update_revocation_registry(
```

Comment:
@andrewwhitehead

Reply:
I think this method will want `cred_def: &CredentialDefinition` and `rev_reg_priv: &RevocationRegistryPrivate` arguments. It's not necessary to create a tails reader. This method behaves similarly: https://github.com/hyperledger/indy-shared-rs/blob/8f51b31039dd0eff1443d3ed3872addcf0958e4c/indy-credx/src/services/issuer.rs#L247